cmd/query/app/querysvc/adjuster/resourceattributes_test.go

```diff
@@ -25,10 +26,10 @@ func TestResourceAttributesAdjuster_SpanWithLibraryAttributes(t *testing.T) {
 	span.Attributes().PutStr("another_key", "another_value")
 
 	adjuster := ResourceAttributes()
-	result, err := adjuster.Adjust(traces)
+	err := adjuster.Adjust(traces)
 	require.NoError(t, err)
```

Comment:
always better not to leak variable into outer scope if possible:
```suggestion
	require.NoError(t, adjuster.Adjust(traces))
```